src/middleware/validations/login.js

```diff
@@ -0,0 +1,24 @@
+import isEmpty from 'lodash/isEmpty';
+
+export default function validateInput(data) {
+  const errors = {};
+  const emailRegex = /^[a-zA-Z0-9.!#$%&'*+/=?^_`{|}~-]+@[a-zA-Z0-9](?:[a-zA-Z0-9-]{0,61}[a-zA-Z0-9])?(?:\.[a-zA-Z0-9](?:[a-zA-Z0-9-]{0,61}[a-zA-Z0-9])?)*$/;
+  const pwdRegex = /^(?=.*?[A-Z])(?=(.*[a-z]){1,})(?=(.*[\d]){1,})(?=(.*[\W]){1,})(?!.*\s).{8,}$/;
+
+  if (data.email === '') {
+    errors.email = 'Email is required';
+  } else if (emailRegex.test(data.email) === false) {
+    errors.email = 'Email is invalid';
+  }
+
+  if (data.password === '') {
+    errors.password = 'Password is required';
+  } else if (pwdRegex.test(data.password) === false) {
```

Comment:
@Hervera Kindly don't validate the password, you must return email or password is not correct.